lib/index.ts

```diff
@@ -88,27 +83,7 @@ export interface DatadogMonitorThresholds {
     readonly warningRecovery?: number;
 }
 
-export interface DatadogCredentials {
-    /**
-     * Datadog API key
-     */
-    readonly apiKey: string;
-
-    /**
-     * Datadog application key
-     */
-    readonly applicationKey: string;
-
-    /**
-     * Datadog API URL.
-     * Caution, we default to https://api.datadoghq.eu, use https://api.datadoghq.com for US accounts.
-     */
-    readonly apiUrl?: string;
```

Comment:
should we keep this option?

Reply:
yes we should :)

Reply:
The API URL should be configured with the credentials directly in the extension:

![Screenshot 2024-02-29 at 6 56 20 PM](https://github.com/avrios/cdk-datadog/assets/44823/287cfd83-3e5a-49cf-861c-ee641ce664de)

Reply:
yeah makes sense to have this all specified centrally 👍